base.go

```diff
@@ -21,17 +21,11 @@ type BaseObject struct {
 	o *C.PyObject
 }
 
-var baseObjMap = make(map[*C.PyObject]*BaseObject)
```

Comment:
I don't get why are you removing this? and similar ones.

Reply:
it causes memory leaks. there is no code which in charge to remove an item from the map

Reply:
But doesn't this enable us to reuse an object created before and not create it again?

Reply:
I think its broken by design the way to save few bytes and couple milliseconds :) and caused a lot of memory leaks with random data. There are two ways to fix it:

1) remove the reason (pull request)
2) implement some kind of TTL (time to live) for every single item stored this way

its your repo, your choice :)

Anyway, i've forked this repo and use it without leaks

Reply:
Ok, so I will check if there is any problem with other repositories while using these changes. Thanks